examples/rbd/storageclass.yaml

```diff
@@ -9,34 +9,38 @@ provisioner: rbd.csi.ceph.com
 # For further information read TODO<doc>
 # volumeBindingMode: WaitForFirstConsumer
 parameters:
-   # String representing a Ceph cluster to provision storage from.
+   # (required) String representing a Ceph cluster to provision storage from.
    # Should be unique across all Ceph clusters in use for provisioning,
    # cannot be greater than 36 bytes in length, and should remain immutable for
    # the lifetime of the StorageClass in use.
    # Ensure to create an entry in the configmap named ceph-csi-config, based on
    # csi-config-map-sample.yaml, to accompany the string chosen to
    # represent the Ceph cluster in clusterID below
    clusterID: <cluster-id>
-   # If you want to use erasure coded pool with RBD, you need to create
-   # two pools. one erasure coded and one replicated.
+
+   # (optional) If you want to use erasure coded pool with RBD, you need to
+   # create two pools. one erasure coded and one replicated.
    # You need to specify the replicated pool here in the `pool` parameter, it is
    # used for the metadata of the images.
    # The erasure coded pool must be set as the `dataPool` parameter below.
-   # dataPool: ec-data-pool
-   pool: rbd
+   # dataPool: <ec-data-pool>
+
+   # (required) Ceph pool into which the RBD image shall be created
+   # eg: pool: rbdpool
+   pool: <rbd-pool-name>
 
-   # RBD image features, CSI creates image with image-format 2
+   # (optional) RBD image features, CSI creates image with image-format 2
```

Comment:
This is not optional. we need to have image Features. I have seen an issue in mapping the rbd image if the images are created without any image features as ceph add few features by default @pkalever can you please test it out.

Reply:
@Madhu-1 I agree with you, let me test this, please add the DNM label until I get back.